docs/articles/vs-test-adapter/AdapterV2-Release-Notes.md

```diff
@@ -21,7 +21,7 @@ Thanks to [BobSilent](https://github.com/BobSilent) for [#188](https://github.co
 
 ### Features
 
-* [#180](https://github.com/nunit/nunit-vs-adapter/issues/180) Nunit 2 test adapter does not support Visual Studio 2019
```

Comment:
Should you be changing release notes that were already published?

Reply:
I thought about this at the time. I decided that capitalization changes didn't substantively alter the nature of the artifacts or documentation, much as correcting a typo wouldn't.